homeassistant/components/crownstone/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""Integration for Crownstone."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .entry_manager import CrownstoneEntryManager
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Initiate setup for a Crownstone config entry."""
+    manager = CrownstoneEntryManager(hass, entry)
+
+    if not await manager.async_setup():
+        return False
+
+    return True
```

Comment:
```suggestion
    return await manager.async_setup()
```

Looks like this already returns a bool